code/src/header/client_udp.hpp

```diff
@@ -118,9 +147,42 @@ void Client_udp::handleHeartbeat(struct addr_info& addr)
             cout<<"CONNECTION LOST!!"<<endl;
             throw "Connection lost";
         }
-        
+
         cout<<"\tHeartbeat response received: "<<buffer<<endl; 
 
         sleep(8);
     }
 }
+
+void Client_udp::process_message(struct message& msg)
+{
+    struct message msg;
```

Comment:
Zmienna przesłania parametr/argument funkcji. Kompilator wyrzuca błąd. Chyba nie o to chodziło